playbooks/playbook_tomcat.yml

```diff
@@ -1,41 +1,12 @@
 ---
 - name: "Validate Common Criteria recommendations on Tomcat installation"
   hosts: all
-  become: yes
-  become_user: root
   vars_files:
-    - vars.yml
-  collections:
-    - middleware_automation.jws
-    - middleware_automation.common_criteria
-  roles:
-    - jws
-    - jws_common_criteria
+    - ../molecule/tomcat/vars.yml
```

Comment:
It's a bit weird to have the "demo" playbook import a set of vars from the testsuite. I would suggest to create a vars/ folder inside playbooks/ and move this file there (so playbooks/vars/tomcat.yml and same for wildfly.yml).

Reply:
Thanks! done